features/lwipstack/mbed_lib.json

```diff
@@ -114,7 +114,7 @@
         },
         "tcpip-thread-stacksize": {
             "help": "Stack size for lwip TCPIP thread",
-            "value": 1200
+            "value": 2048
```

Comment:
This change is not related to NVIC change, please send it  in separate PR

Reply:
Without this change, the device can't register with cloud. Therefore, I can't really test all code changes with confidence. What if I change the title to reflect this as well?

Reply:
@linlingao This will affect all targets. If you want to limit the scope of this change to just the dragonfly you can use a `target_override` instead. Let me know if you want some pointers on how to go about doing that.

Reply:
+1 for override
Taken from targets.json file, some targets overwrite this one:

```
  128      "target_overrides": {
  129          "REALTEK_RTL8195AM": {
  130:             "tcpip-thread-stacksize": 1600,
  131              "mem-size": 12800
  132          },
  ...
```